tests/bugfixes/github/test_issue_981.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+
+from system_tests import CaseMeta, path
+
+class CanonAfInfoTest(metaclass=CaseMeta):
+
+    filenameA = path("$data_path/test_issue_981a.exv")
+    filenameB = path("$data_path/test_issue_981b.exv")
+    filenameC = path("$data_path/test_issue_981c.exv")
+    filenameC = path("$data_path/test_issue_981c.exv")
+    filenameD = path("$data_path/test_issue_981d.exv")
+    commands  = ["$exiv2 -pa --grep Canon.AF $filenameA"
+                ,"$exiv2 -pa --grep Canon.AF $filenameB"
```

Comment:
Can you please move the commas to the end of the preceding lines? I usually don't like to argue about style & formatting, but that is against every Python style guide that ever existed.

Reply:
I'm not going to argue.